enemies.yaml

```
@@ -0,0 +1,76 @@
+# Generic enemies
+Goblin:
+  plural: "Goblins"
+  damage:
+    min damage: 1
+    max damage: 5
+  health:
+    min spawning health: 3
+    max spawning health: 8
+    max health level: 8
+  agility: 0.7
+  inventory:
+  - Knife
+  - Beef
+  - Chicken
+  - Raw Salmon
+  - Bone
+  - Wooden Stick
+  - Orc Iron Leggings
+  - Orc Shredded Chestplate
+  description: "The Goblin may be an easy enemy to beat, but in groups, they can be very annoying to deal with."
+Orc:
+  plural: "Orcs"
+  damage:
+    min damage: 1
+    max damage: 4
+  health:
+    min spawning health: 4
+    max spawning health: 6
+    max health level: 6
+  agility: 0.95
+  inventory:
+  - Knife
+  - Raw Salmon
+  - Bone
+  - Spear
+  - Dagger
+  - Orc Iron Leggings
+  - Orc Shredded Chestplate
+  - Orc Bow
+  description: "Orcs are the smaller and swifter version of the Goblins. Even if they have low health, they are fast and hard to hit because of their agility."
```

A few things:

1) In Lord of the Rings, Yrch = Goblins.

```suggestion
  description: "Yrch are the smaller and swifter version of the Goblins. Even if they have low health, they are fast and hard to hit because of their agility."
```

2) I would actually switch Yrch and Goblins around, making Yrch the stronger and less agile.

---

Fixed by commit: [2106923](https://github.com/lumbar527/Dungeons-of-Kathallion/pull/19/commits/210692351527cbde6ba1d212949d5f3469a34cc1)